buildSrc/src/main/groovy/com/google/firebase/gradle/plugins/license/LicenseResolverPlugin.groovy

```diff
@@ -98,7 +98,7 @@ class LicenseResolverPlugin implements Plugin<Project> {
 
                 // must run before preBuild, otherwise it produces an empty license report.
                 project.tasks.getByName('preBuild').mustRunAfter licensesTask
-                project.tasks.getByName("bundle${variantName}") {
+                project.tasks.getByName("bundle${variantName}Aar") {
```

Comment:
@ashwinraghav can you sanity check this?
@iamyaoxi can you explain this change?  Is this something to do with the newer gradle plugin?

Reply:
Yes. I think it's related to the newer Gradle plugin. I tried to use bundle${variantName} task, it always failing. And I read it's already replaced with bundleAar

Reply:
SGTM

Reply:
:)